crates/bin/c8y-device-management/src/main.rs

```diff
@@ -68,6 +69,10 @@ async fn main() -> anyhow::Result<()> {
     log_actor.with_c8y_http_proxy(&mut c8y_http_proxy_actor)?;
     log_actor.with_mqtt_connection(&mut mqtt_actor)?;
 
+    //Instantiate health monitor actor
+    let mut health_actor = HealthMonitorBuilder::new();
+    health_actor.with_mqtt_connection(&mut mqtt_actor)?;
```

Comment:
```suggestion
    health_actor.set_connection(&mut mqtt_actor)?;
```
Although there's nothing wrong with this, the recommended way is to use `ServiceConsumer::connect_to` method instead of custom methods like this. We want to promote the usage of that API more to make the connection part standardised for all actors.